packages/autocomplete/src/Autocomplete.stories.tsx

```diff
@@ -64,3 +63,17 @@ Block.args = {
 	...defaultArgs,
 	block: true,
 };
+
+export const ExternalAPI = Template.bind({});
+ExternalAPI.args = {
+	...defaultArgs,
```

Comment:
```suggestion
	...defaultArgs,
		hideOptionalLabel: true,
		label: 'Pick a Star Wars character',
```
<img width="419" alt="image" src="https://user-images.githubusercontent.com/12689383/209247564-59574e67-4288-4787-a347-c4e2c9510b73.png">